City-of-Helsinki/drupal-helfi-rekry
# UHF-10341 job page cache problem

# [UHF-10341](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10341)
Joblisting start date cached as "today" and after a week, it would still say "today"

## What was done
Moved the logic from template to javascript.
- Created javacsript to handle the "today" text
- Removed the old implementation from the template and code.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10341`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Create a new job listing, set the startday or publish day or whatever to today
- Save job listing
- Load the job listing page
  - The javascript should load
  - The datetime in metadata-box under julkaistu-title should turn to "today" 




[UHF-10341]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10341?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
